composer.json

```diff
@@ -28,7 +28,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^8.0"
+        "php": "^8.1"
```

Comment:
Should we raise development dependencies versions also?

Reply:
Dependency versions use the latest versions.

phpunit 10 with PHP 8.1 version requirements still in development